source/module_io/read_input_item_exx_dftu.cpp

```diff
@@ -149,6 +149,18 @@ void ReadInput::item_exx()
         read_sync_double(input.exx_v_grad_threshold);
         this->add_item(item);
     }
+    {
+        Input_Item item("exx_c_grad_r_threshold");
```

Comment:
could you change 'threshold' to 'thr' to be consistent with other vaiables?

Reply:
another issue is the input variable name may be a bit too long, is it possible to be shorter?